docs/docs/software-architecture.md

```diff
@@ -38,9 +38,9 @@ The report will present a detailed analysis of the architecture of a platform en
 
 ## 2. Architectural Represantition
 
-Swapchain offers a web application that follows the Clean architecture pattern. Main reason to use this pattern is to separate functions into layers thus improve the maintainability and reusability. Figure 1 visualizes the Clean Architecture by using color-coded schemes.
+Swapchain offers a command-line interface (CLI) and a basic mockup which follow the Clean Architecture pattern. Main reason to use this pattern is to separate functions into layers and thus, improve the maintainability and reusability. Figure 1 visualizes the Clean Architecture by using color-coded schemes.
```

Comment:
Do we only provide a mockup or an actual basic user interface?

Reply:
Done